packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/BYOCWrapper.tsx

```diff
@@ -0,0 +1,15 @@
+import { BYOCProps, BYOCRenderer } from '@sitecore-jss/sitecore-jss-nextjs';
+import React from 'react';
+import * as FEAAS from '@sitecore-feaas/clientside/react';
+
+export const Default = (props: BYOCProps): JSX.Element => {
+  const rendererProps = {
+    components: FEAAS.External.registered,
+    ...props,
+  };
+  return (
+    <div className="component-content">
```

Comment:
For full SXA support, would we need to include the outer wrapping div with rendering parameter id and styles as well (e.g. like we have in [FEaaSWrapper.tsx](https://github.com/Sitecore/jss/blob/dev/packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/FEaaSWrapper.tsx#L16))?

Reply:
I haven't considered this, I'll ask the DevEx team if this is something they are adding in rendering parameters and then add another div wrapper.